tools/mail.py

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 # This script has moved.
-management/cli.py "$@"
+../management/cli.py "$@"
```

Comment:
```suggestion
$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../management/cli.py "$@"
```

This solves the problem of cwd.  I believe this is idiomatic bash for getting the directory in which a script resides.